.devcontainer/Dockerfile

```diff
@@ -79,7 +79,7 @@ COPY ["airlock_processor/requirements.txt", "/tmp/pip-tmp/airlock_processor/"]
 RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/requirements.txt && rm -rf /tmp/pip-tmp
 
 # Install azure-cli
-ARG AZURE_CLI_VERSION=2.36.0-1~buster
+ARG AZURE_CLI_VERSION=2.37.0-1~buster
```

Comment:
IIRC the version is also pinned in the resource processor image and think those should be in-sync.

Reply:
Many thanks! Done.